thrift/src/main/thrift/entity.thrift

```diff
@@ -0,0 +1,26 @@
+namespace java com.gu.contententity.thrift.entity
+
+include "entities/person.thrift"
+include "entities/film.thrift"
+include "entities/game.thrift"
+include "entities/restaurant.thrift"
+
+enum EntityType {
+  PERSON = 0
+  FILM = 1
+  GAME = 2
+  RESTAURANT = 3
+}
+
+// A container for an entity
+struct Entity {
+  1: required string id         //Unique entity ID, e.g. person/theresa_may/12345
+  2: required EntityType entityType
+  3: optional string tagId      //Guardian Tag ID
```

Comment:
Its horrible, but you can generate a large map of tag ids to entity ids when you create the entities. That can be saved to file and then loaded into memory in porter and used as a lookup when adding the entity ids to tags in the second phase. Then this can subsequently be removed.